pkg/kubelet/kubelet.go

```diff
@@ -412,7 +412,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	seccompDefault bool,
 ) (*Kubelet, error) {
 	ctx := context.Background()
```

Comment:
```suggestion
	ctx := context.TODO()
```
and comment it out, please.